src/preamble.js

```diff
@@ -745,7 +746,7 @@ function instrumentWasmTableWithAbort() {
   // Override the wasmTable get function to return the wrappers
   var realGet = wasmTable.get;
   wasmTable.get = function(i) {
-    return makeWrapper(realGet.call(wasmTable, i));
+    return makeAbortWrapper(realGet.call(wasmTable, i), i);
```

Comment:
I see we might call get(3) many times and expect to get the same cached result each time.

In that case maybe just move the cache down here and only do it for table slots?      

Or you could make the second argument to `makeAbortWrapper` and only trigger caching when it is defined (and don't pass it above).

Reply:
Yeah, I moved it in there, it's cleaner that way :)